client/web/compose/src/components/ModuleFields/Configurator/Select.vue

```diff
@@ -10,14 +10,14 @@
           class="mb-1"
         >
           <b-form-input
-            v-model="f.options.options[index].value"
+            v-model.trim="f.options.options[index].value"
```

Comment:
Only value should be trimmed here, text can have spaces since we just display that